DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ConferenceCell.java

```diff
@@ -0,0 +1,174 @@
+import javafx.concurrent.Task;
+import javafx.css.PseudoClass;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Optional;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class ConferenceCell extends CharmListCell<Conference> {
+
+    private static final Logger LOG = Logger.getLogger(ConferenceCell.class.getName());
+
+    private static final PseudoClass PSEUDO_CLASS_VOXXED = PseudoClass.getPseudoClass("voxxed");
+    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("MMMM dd, yyyy");
+    
+    private static final ExecutorService executor = Executors.newCachedThreadPool();
+    
+    private final Service service;
+    private final Label name;
+    private final Label eventType;
+    private final Label dateLabel;
+
+    private final ImageView background;
+    private final VBox top;
+    private final BorderPane content;
+    private final StackPane root;
+    private final DoubleBinding widthProperty;
+
+    public ConferenceCell(Service service) {
+        this.service = service;
+
+        widthProperty = MobileApplication.getInstance().getGlassPane().widthProperty().subtract(15);
+
+        name = new Label();
+        name.getStyleClass().add("name");
+        
+        eventType = new Label();
+        eventType.getStyleClass().add("type");
+        eventType.textProperty().addListener((observable, oldValue, newValue) -> {
+            if (newValue.equals(Conference.Type.VOXXED.name())) {
+                pseudoClassStateChanged(PSEUDO_CLASS_VOXXED, true);
+            } else {
+                pseudoClassStateChanged(PSEUDO_CLASS_VOXXED, false);
+            }
+        });
+        
+        dateLabel = new Label();
+        dateLabel.getStyleClass().add("date");
+        
+        background = new ImageView();
+        background.setFitHeight(222);
+        
+        top = new VBox(eventType, name);
+        top.getStyleClass().add("top");
+
+        content = new BorderPane();
+        content.getStyleClass().add("content");
+        content.setTop(top);
+        content.setBottom(dateLabel);
+        
+        root = new StackPane(background, content);
+        getStyleClass().add("conference-cell"); 
+    }
+
+    @Override
+    public void updateItem(Conference item, boolean empty) {
+        super.updateItem(item, empty);
+        
+        if (item != null && !empty) {
+            eventType.setText(item.getEventType().name());
+            
+            name.setText(item.getName());
+            dateLabel.setText(LocalDate.parse(item.getFromDate()).getDayOfMonth() + " - " + 
```

Comment:
Single day events (where from date and end date are at the same day) can show a single date instead of using the separator with the dash?

Reply:
Good idea.